changelog/issue-4034.md

```diff
@@ -0,0 +1,5 @@
+audience: deployers
+level: patch
+reference: issue 4034
+---
+The queue's artifact expiration crontask now uses a much more efficient query and should be able to keep up with the load.
```

Comment:
Do we usually point out new / deprecated functions, or omit it as implementation details?

Reply:
Since it's internal to the services, it doesn't need mention in the changelog.